Removing the function regarding the account window

chongjunkim opened the pull request:
I have commented out the functions regarding the open/close of the account page.
I have also added the function to pull the account number from the launch page itself.
In addition, I've changed the font and background color to make it more.. clean and fit more closely to the Salesforce interface.

Commit:
Removing the function regarding the account window

I have commented out the functions regarding the open/close of the account page.
I have also added the function to pull the account number from the launch page itself.
In addition, I've changed the font and background color to make it more.. clean and fit more closely to the Salesforce interface.

cirept reviewed (commented):
Instead of commenting out sections / pieces of code, could you please remove the code sections entirely.   This will make maintenance easier down the road.  'removing the fluff'

Commit:
Removed the fluff

Removed all the irrelevant code rather than commenting them out as requested!

cirept reviewed (commented):
Everything looks good.  I like what you did with the coloring.  lol.  You went to total opposite way of why I originally colored the toolbar funny, to not blend in with Salesforce.  I wanted to be able to quickly see if people were using it or not from afar.   haha  :]

- The only thing left to do is to test it using a pre-release.
- You would need to update the "launchToolbar_meta.js" file.  The line that says "// @require https://raw.githubusercontent.com/cirept/salesforceToolbar/v4.5/launchToolbar_release.js".  Update the version number to the one you have created via 'releases' tab.

Commit:
Changing versions from v4.5 to v4.6

cirept reviewed (commented):
two (2) things
1. launchToolbar_meta.js
- the name of the toolbar needs to be updated.
- I put that in there for development purposes.  In case you have multiple scripts in tampermonkey.  You can remove the number from the name since we already have the "@version" field.

2. launchToolbar.js
- The "@require" where you update the version.  // @require https://raw.githubusercontent.com/cirept/salesforceToolbar/v4.6/launchToolbar_release.js

- this is a file that has obfuscation code on it.  I experimented with it as that is normally what production does to protect their code.  You can perform the same OR simply link directly to the launchToolbar.js file instead of the launchToolbar_release.js.

Commit:
Changed title and updated @require URL

Removed the version number from the title as requested, and updated the @require URL to the launchToolbar.js. I'll test out the encrypted version in a later time. ;)

Commit:
Changed @require to match the pre-release

I changed "master" to "v4.6.1" to match your pre-release version!

cirept reviewed (commented):
Everything looks good.  Now we just have to test out the pre-release and we should be good to go.

1. Please update the pre-release with a modified version number. (v4.6-pre-release)
- Reason behind this is because every "release" remembers the exact state of the repo when you create it. (I already did this)

Commit:
Changed the @require URL to v4.6

Commit:
Foreseeing the future :o

Commit:
changing to -2

cirept reviewed (approved):
Everything looks good!  Push this live!

cirept commented:
Please publish the pre-release After you merge the branch
